### ISSUE TITLE
Handle the case of no IS in features that require IS to lookup

### ISSUE THREAD

**Issue by jryans** [vector-im/riot-web#10619]
Since it's now possible to be disconnected from an identity server entirely (your session has no active identity server), it's unclear how features that require an IS to function should behave.

For the case of the Discovery section in Settings, it's easy enough: you have no IS, so we don't show any 3PIDs to control.

However, there are other IS-using features in Riot, like:

1. Inviting by 3PID via the address picker

<img width="792" alt="2019-08-23 at 16 02" src="https://user-images.githubusercontent.com/279572/63602530-8c9c7500-c5bf-11e9-8931-ec308e89de78.png">

2. Inviting by 3PID via `/invite bob@bob.com`
3. ~~Checking an email invite in the invite preview bar to ensure the invited email is for the active account (in case you have multiple Matrix accounts in different tabs, etc.)~~ (Extracted feature 3 to https://github.com/vector-im/riot-web/issues/10669.) 

For features 1 and 2, I think we need some UX advice from @nadonomy on what they should do (communicate some kind of error to the user, etc.).

For feature 3, I think we are at an earlier stage where @lampholder needs to work out how the product should function for this case.

**Comment by nadonomy** [comment]
@jryans on these, I just edited these via inspector at runtime, but could we do something along these lines:

**1. Inviting via address picker**

<img width="1792" alt="Screenshot 2019-08-23 at 15 03 48" src="https://user-images.githubusercontent.com/4626865/63626283-55c26100-c5b7-11e9-84ed-5e5d48fab701.png">

- [ ] Remove the 'Who would you like to this room?' text as it's redundant
- [ ] Add helper text below the input. Validation should continue to be the closest to the input, so this helper text should live after all the things that relate to input, and before the buttons.

**2. Inviting via /invite** 
<img width="1792" alt="Screenshot 2019-08-23 at 14 58 18" src="https://user-images.githubusercontent.com/4626865/63626337-94581b80-c5b7-11e9-9215-fd227a9be377.png">

- [ ] Display a generic modal when the user completes the action. We should be able to use this in other places too.


**3. Same text, contextual styles**
- [ ] Same as 2, I think just contextually displaying the same text in place should be fine using the contextual styles you screenshotted.

**Comment by jryans** [comment]
Thanks @nadonomy! There's enough work here for features 1 and 2 that it's seems reasonable to consider at least that portion unblocked.

Since the status of feature 3 is less clear, I have broken it out to a separate issue https://github.com/vector-im/riot-web/issues/10669 which is marked blocked for now.
